ui/server/src/main/scala/pl/touk/nussknacker/ui/api/AppResources.scala

```diff
@@ -50,30 +51,47 @@ class AppResources(config: Config,
       path("healthCheck") {
         get {
           complete {
-            notRunningProcessesThatShouldRun.map[HttpResponse] { set =>
+            Marshal(HealthCheckProcessResponse(OK))
```

Comment:
You can extract this to some method like:

```
  private def healthCheckResponse(response: HealthCheckProcessResponse, statuCode: StatusCode) =
    Marshal(response)
      .to[ResponseEntity]
      .map(res => HttpResponse(status = statuCode, entity = res))

Reply:
You can also do some method maped HealthCheckProcessResponseStatus to StatusCodes and use it here.